compiler/crates/relay-transforms/src/apply_transforms.rs

```diff
@@ -130,6 +134,7 @@ fn apply_common_transforms(
     program: Arc<Program>,
     connection_interface: &ConnectionInterface,
     feature_flags: Arc<FeatureFlags>,
```

Comment:
the benefits of adding `node_interface_id_field` to `FeatureFlags` is that you already have them in `apply_transforms` so it's less changes.